templates/vmset/cloudinit/userdata

```diff
@@ -12,4 +12,8 @@ chpasswd:
 {{- end }}
 bootcmd:
   - "mkdir /mnt/kubeconfig"
-  - "mount /dev/disk/by-id/virtio-fencingdisk /mnt/kubeconfig"
\ No newline at end of file
+  - "mkdir /root/kubeconfig"
+  - "mount /dev/disk/by-id/virtio-fencingdisk /mnt/kubeconfig"
```

Comment:
could also check to use the mounts feature of cloud-init https://cloudinit.readthedocs.io/en/latest/topics/examples.html#adjust-mount-points-mounted

Reply:
I'll take a look

Reply:
Decided to just keep it as-is and add your flag and unmount suggestions